openapi/plugin_config_schema.go

```diff
@@ -53,14 +52,6 @@ func (p *PluginConfigSchemaV1) Validate() error {
 	if p.Version != "1" {
 		return fmt.Errorf("provider configuration version not matching current implementation, please use version '1' of provider configuration specification")
 	}
-	for k, v := range p.Services {
```

Comment:
documenting for future reference: this is a bug fix unrelated to this PR

Reply:
updated the issue description to cover this too